src/components/Cards/Card.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import themeGet from 'extensions/themeGet';
+import DefaultCardHeader from './CardHeader';
+import CardSection from './CardSection';
+
+const OuterBox = styled.div`
```

Comment:
All presentational styling components should be defined in `./styles/` and injected as props.